<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SettingsWindow</class>
 <widget class="QDialog" name="SettingsWindow">
  <property name="windowModality">
   <enum>Qt::NonModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>805</width>
    <height>800</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>IntelliFiller Settings</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="labelApiKey">
     <property name="text">
      <string>OpenAI API Key:</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="apiKey">
     <property name="placeholderText">
      <string>OpenAI API key</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="labelOpenaiModel">
     <property name="text">
      <string>OpenAI Model:</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="openaiModel">
     <property name="placeholderText">
      <string>gpt-4o-mini</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="labelAnthropicKey">
     <property name="text">
      <string>Anthropic API Key:</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="anthropicKey">
     <property name="placeholderText">
      <string>Anthropic API key</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="labelAnthropicModel">
     <property name="text">
      <string>Anthropic Model:</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="anthropicModel">
     <property name="placeholderText">
      <string>claude-haiku-4-5</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="labelGeminiKey">
     <property name="text">
      <string>Google Gemini API Key:</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="geminiKey">
     <property name="placeholderText">
      <string>Google Gemini API key</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="labelGeminiModel">
     <property name="text">
      <string>Google Gemini Model:</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="geminiModel">
     <property name="placeholderText">
      <string>gemini-2.0-flash-lite-001</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="labelSelectedApi">
     <property name="text">
      <string>Selected API:</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QComboBox" name="selectedApi">
     <item>
      <property name="text">
       <string>openai</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>anthropic</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>gemini</string>
      </property>
     </item>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="labelEmulate">
     <property name="text">
      <string>Emulate:</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QComboBox" name="emulate">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <item>
      <property name="text">
       <string>yes</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>no</string>
      </property>
     </item>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="addPromptButton">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>Add Prompt</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QScrollArea" name="scrollArea">
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="scrollAreaWidgetContents">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>779</width>
        <height>605</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="promptsLayout"/>
     </widget>
    </widget>
   </item>
   <item alignment="Qt::AlignRight">
    <widget class="QPushButton" name="saveButton">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>Save</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>